test/initialization.test.js

```diff
@@ -64,20 +64,20 @@ test('Should not throw if registered within different scopes (with and without n
 
   fastify.register(function scopeOne (instance, opts, next) {
     instance.register(fastifyPostgres, {
-      connectionString: 'postgres://postgres@localhost/postgres'
+      connectionString: 'postgres://postgres:postgres@localhost/postgres'
     })
 
     next()
   })
 
   fastify.register(function scopeTwo (instance, opts, next) {
     instance.register(fastifyPostgres, {
-      connectionString: 'postgres://postgres@localhost/postgres',
+      connectionString: 'postgres://postgres:postgres@localhost/postgres',
```

Comment:
maybe we can use `.env` to load those up? I think locally it could be different. It would also be different if Docker is used.

Reply:
It is the same locally as well. I updated to docker commands